NAMESPACE

```diff
@@ -7,6 +7,7 @@ export(browse_github)
 export(browse_github_issues)
 export(browse_github_pat)
 export(browse_github_pulls)
+export(browse_github_token)
```

Comment:
We always call this `auth_token`, so I've created this. The old `browse_github_pat()` is aliased to this. These are workflow-y functions, so this can be a pretty low key thing.